runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningPlugin.java

```diff
@@ -256,6 +258,12 @@ public void onGameObjectDespawned(GameObjectDespawned event)
 		}
 	}
 
+	private void clearDaeyaltHintArrow(GameObject object)
+	{
+		if (client.getHintArrowType() == WORLD_POSITION && client.getHintArrowPoint().equals(object.getWorldLocation()))
+			client.clearHintArrow();
```

Comment:
This project avoids the use of one-line `if` statements. Add braces to the `clearHintArrow()` call.
```suggestion
		if (client.getHintArrowType() == WORLD_POSITION && client.getHintArrowPoint().equals(object.getWorldLocation()))
		{
			client.clearHintArrow();
		}
```